LSP.sublime-settings

```diff
@@ -180,6 +180,13 @@
   // Valid values are "underline" or "box"
   "diagnostics_highlight_style": "underline",
 
+  // Signature help style.
+  // Valid values are "sublime" or "vscode".
+  //
+  // - "sublime": Uses mdpopups faithfully.
+  // - "vscode": Tries to imitate VSCode.
+  "signature_help_style": "vscode",
```

Comment:
Do we want this to be `vscode` by default or `sublime` by default?

Reply:
How about a boolean setting named something like `highlight_active_signature_parameter`?
This might be more helpful for users not familiar with VS Code.

We can merge it set to True and then decide before next release what works best.